### PR TITLE
grid_map_ros pkg added

### PR DESCRIPTION
**Pull request by DaikiMaekawa**

### CODE REVIEW

File: package.list [hunk 1]
```diff
@@ -22,3 +22,4 @@ usb_cam
 viz
 xsens_driver
 ypspur_ros
+grid_map_ros
```
Comment (f-fl0) [inline package.list:25]: Sorry to be picky but I think the packages should be defined in alphabetical order.

**Comment by DaikiMaekawa** [comment]
This package is what originally I wanted to make it available on the CI.

File: package.list (hunk 1 above)
Comment (DaikiMaekawa) [reply package.list:25]: Sure, will do.

**Comment by DaikiMaekawa** [comment]
@at-wat @f-fl0 Please take a look at this.

**Review by f-fl0 — APPROVED** [review]
LGTM

**Pull request merged**